pkg/common/versions/installselectors/default_version.go

```diff
@@ -0,0 +1,25 @@
+package installselectors
```

Comment:
Just to confirm, the plan is to use these selectors against version lists in order to select a version?

Reply:
Correct.

Reply:
Coolio